api/docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+info()
+{
+    echo "Info: $*"
+}
+
+welcome_message(){
+  info 'Starting CHT API'
+}
+
+main(){
+  welcome_message
+  node /api/server.js
+}
+
+"$@"
```

Comment:
This feels like overkill when it could just be inlined into the Dockerfile. Is there a reason to have a separate entrypoint script?

Reply:
I also took this from Hareet's setup: https://github.com/medic/cht-core/blob/dockerize_api/api/docker-entrypoint.sh
I think it's useful if we want to add more actions in this script.